go.mod

```diff
@@ -7,8 +7,8 @@ require (
 	cloud.google.com/go/bigquery v1.0.1
 	cloud.google.com/go/bigtable v1.0.0
 	github.com/go-test/deep v1.0.3
-	github.com/golang/protobuf v1.3.4
-	github.com/google/go-cmp v0.3.1 // indirect
+	github.com/golang/protobuf v1.4.0-rc.4
```

Comment:
For my education, when do we update those and where to find the right version?

Reply:
This is automatically updated when I run "go test" locally. I think there is a way to force update this. Can check with Wei. In general, don't really need to update the package.

Reply:
Got it. Thanks